app/views/users/new.html.erb

```diff
@@ -3,7 +3,9 @@
 
   <% if @user.errors.present? %>
     <div class="errors">
-      Please fill out all fields
+      <% @user.errors.full_messages.each do |error_message| %>
+        <%= error_message %>
+      <% end %>
```

Comment:
This is great, but I wonder if we want to exclude the error message that says the email address is already taken. It feels like it might be responsible not to expose that information to anyone trying to use the sign up form. I think there are a few ways we could do this:

1. We can pair the errors with their respective fields (e.g., errors for first name next to the first name field and so on).
2. We might be able to filter out that error in particular.
3.  We can stick to the original generic error, and modify it a bit:
```
All fields are required. If you're having trouble, make sure you don't already have an account with the same email. Reset your password here
```
And we could link them to the password reset form in that message.

I'm partial to 1 or 2, but 3 is super easy. Thoughts?